glide.yaml

```diff
@@ -81,6 +81,8 @@ import:
   version: v3
 - package: google.golang.org/grpc
   version: v1.16.0
+- package: golang.org/x/net
+  version: ab555f366c4508dbe0802550b1b20c46c5c18aa0
```

Comment:
I am considering being a chicken and
a) bumping golang.org/x/net to the minimum commit needed
b) not doing a wider `glide up`